recipes/hasasia/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "hasasia" %}
+{% set version = "1.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 87ab202bb6c07b24922169fc12115f9fe074c9d6ae33df08b91cb3c4ca48436f
+
+build:
+  noarch: python
+  number: 0
+  skip: True  # [py<35]
```

This looks like pure Python so could be noarch Python (and reduce the number of builds):
```suggestion
  noarch: python
```